codegenerator/cli/npm/envio/src/FetchState.res

```diff
@@ -1226,3 +1226,21 @@ let isReadyToEnterReorgThreshold = (
   } &&
   queue->Utils.Array.isEmpty
 }
+
+let hasBatchItem = ({queue, latestFullyFetchedBlock}: t) => {
+  switch queue->Utils.Array.last {
+  | Some(item) => item.blockNumber <= latestFullyFetchedBlock.blockNumber
+  | None => false
+  }
+}
+
+let compareUnorderedBatchChainPriority = (a: t, b: t) => {
+  // Use unsafe since we filtered out all queues without batch items
```

Comment:
```suggestion
/** FetchStates must have items in their queues or there will be a runtime exn */
let compareUnorderedBatchChainPriority = (a: t, b: t) => {
  // Use unsafe since we filtered out all queues without batch items
```

Reply:
Solved it differently